### PR TITLE
chore(Textarea): migrate Textarea to Tailwind

### PR DESCRIPTION
**Pull request by mainframev**
[Closes 1697](https://kiwicom.atlassian.net/browse/FEPLT-1697)
 Storybook: https://orbit-mainframev-tw-migrate-textarea.surge.sh

### CODE REVIEW

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (DSil) [inline packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx:96]: It should not be the same shadow on hover and on normal state

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (DSil) [inline packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx:123]: I think we can avoid this and use tailwind classes

`resize === RESIZE_OPTIONS.VERTICAL ? "resize-y" : "resize-none"`

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (DSil) [inline packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx:94]: Font size does not change with size. They were using fontSizeInputSmall and fontSizeInputNormal, but, as we observed, they were actually the same font-size... So it ahould just be `text-form-element-normal`, regardless of element size.

What's weird to me is that the line-height and padding also change. But maybe that's a bug...

**Review by DSil — CHANGES_REQUESTED** [review]

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (mainframev) [reply packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx:96]: good, solved in #4047, will add the class for now, but will work after the merge of Radio

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (mainframev) [reply packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx]: Waiting for designers reply, might be not necessary to support small size at all

File: packages/orbit-tailwind-preset/src/presets/components/index.ts
Comment (DSil) [inline packages/orbit-tailwind-preset/src/presets/components/index.ts:120]: Please fix this

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (mainframev) [reply packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx]: So we should have only normal-sized textarea

Comment (DSil) [reply packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx]: Give that it's a breaking change, I'd add it to this release right now (aka skip the tmp folder) and include it on this PR. WDYT?

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (DSil) [reply packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx]: This is still not applied

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (mainframev) [reply packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx]: Nope, I won't do that right now, I will prefer it to be released with other input components

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (mainframev) [reply packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx]: Should be ok now :)

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (DSil) [reply packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx:104]: Just realized this suggestion is messy… I changed line 96 and 103 on the original file removing the hover effects from the `error` condition and adding them inside an extra condition inside the `disabled || readOnly` condition

File: packages/orbit-tailwind-preset/src/presets/components/index.ts
Comment (mainframev) [reply packages/orbit-tailwind-preset/src/presets/components/index.ts:232]: You are right, actually, to me, it even visually looks better :) As it's not almost visible, but I will change that because of consistency

Comment (DSil) [reply packages/orbit-tailwind-preset/src/presets/components/index.ts]: I kinda agree. I don't like any of them, tbh. One looks odd to me (two shades of red) and the other is barely noticeable. But that's something we can address designers in the future and easily change the tokens accordingly 🙂

File: packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx
Comment (DSil) [inline packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx:104]: This can't be like this. This is saying the error shadow only appears when it's not disabled nor readOnly, which is not true. Only hover shadow is affected disabled || readOnly.

The suggestion [here](https://github.com/kiwicom/orbit/pull/4053#discussion_r1345877876) has it correctly. It adds the the shadow depending on error state (line 107) and then the hover shadow depends on the disabled || readOnly (line 114).

Comment (mainframev) [reply packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx:104]: it can be disabled and have the error shadow?

**Review by DSil — APPROVED** [review]

**Pull request merged**
